techniques/failures/F81.html

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Failure of Success Criterion 1.4.1 due to identifying required or error fields using color differences only</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></link></head><body><h1>Failure of Success Criterion 1.4.1 due to identifying required or error fields using color differences only</h1><section class="meta"><p class="id">ID: F81</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
       <p>All technologies</p>
    </section><section id="description"><h2>Description</h2>
-      <p>This objective of this technique is to describe the failure that occurs when a required field or an error field is marked with color differences only, without an alternate way to identify the required field or error field. This can cause problems for people who are blind or colorblind, because they may not be able to perceive the color differences that indicate which field is required or which field is causing an error.</p>
+      <p>This objective of this technique is to describe the failure that occurs when a required field or an error field is marked with color differences only, without an alternate way to identify the required field or error field. This can cause problems for people who colorblind, because they may not be able to perceive the color differences that indicate which field is required or which field is causing an error.</p>
```

Comment:
There's a verb missing @JAWS-test2 

> This can cause problems for people who colorblind

to

> This can cause problems for people who **are** colorblind